rholang/src/main/scala/coop/rchain/rholang/interpreter/Interpreter.scala

```diff
@@ -61,7 +61,7 @@ object Interpreter {
                       case Right(parsed) =>
                         for {
                           result    <- reducer.inj(parsed).attempt
-                          phlosLeft <- C.get
+                          phlosLeft <- C.inspect(identity)
```

Comment:
What is the difference?

Reply:
`get` can be blocking if the underlying `MonadState` is an MVar